server/session.js

```diff
@@ -0,0 +1,120 @@
+const _ = require('lodash');
+
+const fetch = require('node-fetch');
+const args = require('./args');
+
+/**
+ * A barebones, minimal "session store" needed to get stuff working. Your actual solution
+ * will obviously need to be more complex than this.
```

Comment:
NP: i think we should drop `obviously` from this to make our tone a little nicer 😸